### ISSUE TITLE
(eval):typeset:1: not valid in this context after installation

### ISSUE THREAD

**Issue by BjoernDaase** [romkatv/powerlevel10k#59]
I have a powerelvel9k setup on Arch.

I wanted to try out powerlevel10k, so I executed
```
git clone https://github.com/romkatv/powerlevel10k.git /tmp/powerlevel10k
source /tmp/powerlevel10k/powerlevel10k.zsh-theme
```
But that results in my terminal showing these three lines
```
(eval):typeset:1: not valid in this context: -g
(eval):typeset:1: not valid in this context: -g
(eval):typeset:1: not valid in this context: -a
```

Besides that, it feels like powerlevel10k is running fine.

**Comment by romkatv** [comment]
What does `echo $ZSH_VERSION` say?

Can you reproduce this in a clean environment? Like this:

```zsh
git clone https://github.com/romkatv/powerlevel10k.git /tmp/powerlevel10k
echo 'source /tmp/powerlevel10k/powerlevel10k.zsh-theme' >/tmp/powerlevel10k/.zshrc
env -i TERM=$TERM LANG=$LANG ZDOTDIR=/tmp/powerlevel10k zsh
```

**Comment by BjoernDaase** [comment]
> What does `echo $ZSH_VERSION` say?

5.7.1


> Can you reproduce this in a clean environment? Like this:
> 
> ```shell
> git clone https://github.com/romkatv/powerlevel10k.git /tmp/powerlevel10k
> echo 'source /tmp/powerlevel10k/powerlevel10k.zsh-theme' >/tmp/powerlevel10k/.zshrc
> env -i TERM=$TERM LANG=$LANG ZDOTDIR=/tmp/powerlevel10k zsh
> ```

No, that does NOT result in an error!

**Comment by romkatv** [comment]
> No, that does NOT result in an error!

Could you try to figure out what you need to add to that code snippet to reproduce this error? The first suspect is the set of `POWERLEVEL9K` options you have. If that doesn't trigger the error, perhaps you'll need to source p9k before sourcing p10k? Or maybe it's one of the plugins you have?

If you can tell me how to reproduce this, I'll be happy to debug and fix the issue.

**Comment by BjoernDaase** [comment]
> > No, that does NOT result in an error!
> 
> Could you try to figure out what you need to add to that code snippet to reproduce this error? The first suspect is the set of `POWERLEVEL9K` options you have. If that doesn't trigger the error, perhaps you'll need to source p9k before sourcing p10k? Or maybe it's one of the plugins you have?
> 
> If you can tell me how to reproduce this, I'll be happy to debug and fix the issue.

Sure, I'll try too.
What might be interesting to add is, that I did not change any p9k/p10k/.zshrc configuration and did not add any plugin.

**Comment by BjoernDaase** [comment]
Btw, if I purge p9k completely and use p10k only (installed via AUR and this line added to my .zshrc 
`source /usr/share/zsh-theme-powerlevel10k/powerlevel10k.zsh-theme`)
the issue gets even worse and looks like this
![Screenshot from 2019-04-22 16-47-51](https://user-images.githubusercontent.com/24682367/56506329-6a65ee00-651e-11e9-8059-dc8edf7e1b77.png)

**Comment by BjoernDaase** [comment]
I found the problem: 
My .zshrc had the following line by default:
`setopt rcexpandparam                                            # Array expension with parameters`
Removing that results in p10k working perfectly.

**Comment by romkatv** [comment]
Thanks! I've reproduced this. Working on a fix.

**Comment by romkatv** [comment]
Fixed in https://github.com/romkatv/powerlevel10k/commit/0e0132e8fad785c87160f39b5d450a28295ee369.

Thanks for the bug report!

**Comment by BjoernDaase** [comment]
> Fixed in [0e0132e](https://github.com/romkatv/powerlevel10k/commit/0e0132e8fad785c87160f39b5d450a28295ee369).
> 
> Thanks for the bug report!

Thank you, works like a charm now :)
